tests/queries/0_stateless/02023_storage_filelog.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+set -eu
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+# Data preparation.
+# Now we can get the user_files_path by use the table file function for trick. also we can get it by query as:
+#  "insert into function file('exist.txt', 'CSV', 'val1 char') values ('aaaa'); select _path from file('exist.txt', 'CSV', 'val1 char')"
+user_files_path=$(clickhouse-client --query "select _path,_file from file('nonexist.txt', 'CSV', 'val1 char')" 2>&1 | grep Exception | awk '{gsub("/nonexist.txt","",$9); print $9}')
+
+mkdir -p ${user_files_path}/logs/
+echo  1, 1 >> ${user_files_path}/logs/a.txt
+echo  2, 2 >> ${user_files_path}/logs/a.txt
+echo  3, 3 >> ${user_files_path}/logs/a.txt
+echo  4, 4 >> ${user_files_path}/logs/a.txt
+echo  5, 5 >> ${user_files_path}/logs/a.txt
+echo  6, 6 >> ${user_files_path}/logs/a.txt
+echo  7, 7 >> ${user_files_path}/logs/a.txt
+echo  8, 8 >> ${user_files_path}/logs/a.txt
+echo  9, 9 >> ${user_files_path}/logs/a.txt
+echo  10, 10 >> ${user_files_path}/logs/a.txt
+
+${CLICKHOUSE_CLIENT} --query "drop table if exists file_log;"
+${CLICKHOUSE_CLIENT} --query "create table file_log(k UInt8, v UInt8) engine=FileLog('logs', 'CSV');"
+
+${CLICKHOUSE_CLIENT} --query "select * from file_log order by k;"
+
+echo  100, 100 >> ${user_files_path}/logs/a.txt
+
+${CLICKHOUSE_CLIENT} --query "select * from file_log order by k;"
+
+cp ${user_files_path}/logs/a.txt ${user_files_path}/logs/b.txt
+
+# touch does not change file content, no event
+touch ${user_files_path}/logs/a.txt
+
+${CLICKHOUSE_CLIENT} --query "select * from file_log order by k;"
+
+${CLICKHOUSE_CLIENT} --query "drop table if exists mv;"
+${CLICKHOUSE_CLIENT} --query "create Materialized View mv engine=MergeTree order by k as select * from file_log;"
+
+cp ${user_files_path}/logs/a.txt ${user_files_path}/logs/c.txt
+cp ${user_files_path}/logs/a.txt ${user_files_path}/logs/d.txt
+# MOVE_FILE_FROM and MOVE_FILE_TO monitored
+mv ${user_files_path}/logs/a.txt ${user_files_path}/logs/e.txt
+rm ${user_files_path}/logs/e.txt
+
+for i in 1 2 3 4 5
+do
+	sleep 1
+done
+
+${CLICKHOUSE_CLIENT} --query "select * from mv order by k;"
+
+echo  111, 111 >> ${user_files_path}/logs/c.txt
+
+for i in 1 2 3 4 5
+do
+	sleep 1
+done
+
```

Comment:
Since reschedule happened, the time cost to push to view is uncentain, may from 0.5s to 32s, but sleep too long in the test seems unacceptable. @kssenii

Reply:
You can just make a loop in bash, may be a for loop with tiny sleeps to make sure test does not timeout.

Reply:
So, timeout does not judge by the total running time of the test?

Reply:
There is no timeout, just the test must be end with either ok or fail -- this loop should be finite

> may from 0.5s to 32s

It seems it should not this quickly get to large backoff, is it?

Reply:
I don't know why these functinonal tests failed.

Reply:
the test should be ok since it passed in fast test, try to add this tags https://github.com/ClickHouse/ClickHouse/blob/e0ce179aa8e86c7c441a464d7388cd346cc8821b/tests/queries/0_stateless/00600_replace_running_query.sh#L2
and also add tag `long`